docs/Developers/Newcomers_Guide.rst

```diff
@@ -298,7 +253,50 @@ Ask a developer to check if it should be reclassified as `low` difficulty.
       please check the following link:
       `PEP8 Style Guide for Python code <https://www.python.org/dev/peps/pep-0008/>`_
 
-Step 3. Creating a Fork and Testing Your Changes
+Step 3. Verify the Issue Exists
+-------------------------------
+
+You must ensure that the issue you would like to work on exists. You should 
+look through the code or documentations to spot the region where the issue resides. 
+This is an important step while taking up an issue which is often overlooked.
+
+.. important::
+
+
+    * If the issue is a bug in the source code then that shall be confirmed by 
+      executing the relevant code or files. You should also figure out how will you rectify this bug.
+
+    * If the issue is about some discrepency in the documentation, then locate the region in the documentation where it exists. 
+      Find out how it can be removed.
+
+    * If the issue requires some additional functionality to be added, then check whether it 
+      is possible and does not conflict with the existing functionalities.
+
+    * If the issue requires the code to have some comments, then provide comments in plain and 
+      easy-to-understand language.
+
+    * If the issue is some grammatical or spelling error either in comments or documentation, 
+      then figure out how it can be corrected. A wrong grammar or a misspelt word can make things difficult 
```

Comment:
"A wrong grammar" should be "Wrong grammar" - the word "grammar" typically isn't preceded by an article.